### ISSUE TITLE
Mouse & Keyboard Lag on bluetooth!

### ISSUE THREAD

**Issue by icydiafire** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#131]
Hello Chris.
I am using Logitech master MX2 and Logitech Craft keyboard and there is lag when I am using them. do you have a solution?

**Comment by jamesfawcett** [comment]
Are you connected via Bluetooth, or using the unifying adapter? I have the same and there's no lag for me, using a Fenvi T919 or the unifying adapter. I like using the adapter because I can change between operating systems without pressing the button on the bottom of the mouse.

**Comment by AlaKru** [comment]
Having the same issue with Apple mouse lag using the Fenvi.

**Comment by jaknowakowski** [comment]
Check this: https://www.codejourney.net/2017/04/wifi-and-bluetooth-interference-diagnosing-and-fixing/

tldr; 
Bluetooth will interference with wifi 2.4 (on some channels). Try using 5 GHz or adjust router channels.
